docker/application.Dockerfile

```diff
@@ -0,0 +1,22 @@
+#############
+## Stage 1 ##
+#############
+FROM ruby:2.6.6-alpine3.12
+
+RUN apk add --no-cache --update build-base cmake
+
+WORKDIR /usr/src/app
+ENV APP_ENV=production
+ENV RACK_ENV=production
+
+COPY Gemfile Gemfile.lock ./
+
+RUN gem install bundler:2.1.4 && \
+    bundle config set deployment 'true' && \
+    bundle config set without 'development test' && \
+    bundle install
```

Comment:
I don't know how big the `build-base` and `cmake` installs are, but you could consider doing a multi-stage build to only have them installed when doing the bundle install, and then later on copying the built gems into the next stage. It will complicate this Dockerfile a bit though.

Reply:
I actually had this (you'll see it in a previous commit) but for some reason none of the bundler stuff was working, so I simplified. PR welcome though if you can see the issue!